### DISCUSSION TITLE
buddhism/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#9] [General]
# buddhism/todo/

https://shanenull.com/buddhism/todo/

**Comment by shane0** [comment]
zen reader:

“MATTER AND EMPTINESS - SEEING MATTER itself as emptiness produces great wisdom so one does not dwell in birth and death; seeing emptiness as equivalent to matter produces great compassion so one does not dwell in nirvana. Yun-feng”

**Comment by shane0** [comment]
https://github.com/shane0/shane0.github.io/discussions/36#discussioncomment-6180857

**Comment by shane0** [comment]
https://youtu.be/WaBusjvbVT0?si=KWzZ38uSiPdL0DE-

Not self

**Comment by shane0** [comment]
Laser cut cards with heart sutra and maranasati

**Comment by shane0** [comment]
page explaining eyes of buddha

**Comment by shane0** [comment]
**Cormac McCarthy style:**

Stream entry is the first crack in the egg of the ego.

It is the first glimpse of the truth beyond the illusion of self.

It is the first taste of freedom from the suffering of samsara.

Stream entry is not the end of the path, but it is the beginning of the end.

It is the irreversible path to liberation.

Once you have tasted stream entry, you will never be the same.

You will be forever drawn to the truth, even if you do not yet know what it is.

You will be forever striving for liberation, even if you do not yet know how to achieve it.

Stream entry is the first step on the journey to enlightenment.

It is a step that can never be taken back.

So hold on tight, and enjoy the ride.

**Comment by shane0** [comment]
gayatri page with translation explanation etc
https://youtu.be/oZsn5gjMeM4?si=EzGUWpeI6EHhOieH

prajnaparamita page

diamond sutra

**Comment by shane0** [comment]
The Gayatri Mantra is one of the most sacred and powerful mantras in Hinduism. It is a prayer to the Supreme Being, asking for illumination and guidance. The mantra is in Sanskrit, and its English translation is as follows:

**Om Bhur Bhuvah Svah**
**Tat Savitur Varenyam**
**Bhargo Devasya Dhimahi**
**Dhiyo Yo Nah Prachodayat**

This can be translated as:

**Om, the Supreme Being is the Earth, the Atmosphere, and the Heavens.**
**We meditate on the glorious light of that divine Sun.**
**May that divine light illuminate our intellect.**

The Gayatri Mantra is often chanted during meditation and yoga practice. It is also recited at religious ceremonies and rituals. The mantra is believed to have many benefits, including:

* It can help to improve concentration and focus.
* It can help to reduce stress and anxiety.
* It can help to promote spiritual growth and development.
* It can help to increase intelligence and wisdom.

The Gayatri Mantra is a universal prayer that can be chanted by people of all faiths. It is a powerful tool for self-transformation and spiritual growth.

**Comment by shane0** [comment]
https://youtu.be/nEF8C7Ph9Z4?si=XsdebSvbBauKowL1

**Comment by shane0** [comment]
home page buttons
at work at home traveling sitting walking eating etc

**Comment by shane0** [comment]
measured brain effects of meditation https://youtu.be/lF7Qgsp1TPw

**Comment by shane0** [comment]
homepage other button should link to https://shanenull.com/buddhism/core/#the-40-objects-of-meditation-in-buddhism-are

**Comment by shane0** [comment]
https://drive.google.com/drive/folders/1VYTr5l7jARi_Kb_aB0Jjjq2RZF9kacK7

**Comment by shane0** [comment]
- upside down views & garfield & dude in olympia
- real for unreal & mind seal threefold emptiness
- empty of independence or permanence instead it is karmic & spontaneous as is unfolding reality without delusion
- brb

**Comment by shane0** [comment]
https://www.instagram.com/sadhguru/

- gurus are not your friend or teacher they just burn the lights so you see
- transforming religion to responsibility

- i know what i know
- i do not know what i do not know

- there is no place for beliefs you know or you do not know
- science doubts everything and believes in nothing without proof and even then you are encouraged to disprove everything
- belief is a lie it's a way to stop thought and control people and make them infight

**Comment by shane0** [comment]
rewrite a page with precepts ten wholesome deeds do all good etc. page 50

**Comment by shane0** [comment]
use mike dropdown linji dzogzen zen theravada mahayana etc
